### ISSUE TITLE
Problems with logging-1 tests

### ISSUE THREAD

**Issue by ilya8008** [express42/otus-homeworks#2]
There is a problem with test:
```
File logging/fluentd/fluent.conf content should match /pattern service=%{WORD:service} \| event=%{WORD:event} \| request_id=%{GREEDYDATA:request_id} \| message='%{GREEDYDATA:message}'/
```
File fluent.conf is correct.

**Comment by ilya8008** [comment]
I have published PR to resolve this issue.
https://github.com/express42/otus-homeworks/pull/1

**Comment by virtualshuric** [comment]
Fixed in [c857a4c](https://github.com/express42/otus-homeworks/commit/c857a4c3224ebe8c95e91b8d66e790525b7e4541)
PR is unfortunately invalid. 
Thanks for reporting, anyway! It's really nice to get an active feedback!
